lib/locales/pl.yml

```diff
@@ -1,14 +1,49 @@
 pl:
-  figure: Rysunek
-  table_label: Tabela
+  figure:
+    nominative: Rysunek
+    genitive: Rysunku
+    dative: Rysunkowi
+    accusative: Rysunek
+    instrumental: Rysunkiem
+    locative: Rysunku
+    vocative: Rysunku
+  table:
+    nominative: Tabela
+    genitive: Tabeli
+    dative: Tabeli
+    accusative: Tabelę
+    instrumental: Tabelą
+    locative: Tabeli
+    vocative: Tabelo
```

Comment:
Does Polish use all the cases? Otherwise, looks good

Reply:
*the Polish book, the language, sorry

Reply:
Unfortunately yes :smile: Polish language is complicated.